src/pages/products/testing-center/index.tsx

```diff
@@ -0,0 +1,181 @@
+import classnames from "classnames";
+
+import { PageBanner } from "../../../components/Banner";
+import { Heading } from "../../../components/Heading";
+import { BannerCTA } from "../../../components/CallToActionBanner";
+
+import cbtfFormatImage from "../../../lib/images/cbtf.jpg";
+import reducedDistractionImage from "../../../lib/images/reduced-distraction.png";
+
+import styles from "./index.module.scss";
+
+interface ValuePropCardProps {
+  title: string;
+  children: React.ReactNode;
+}
+
+const ValuePropCard: React.FC<ValuePropCardProps> = ({ title, children }) => {
+  return (
+    <article className={classnames("card overflow-hidden")}>
+      <div className="card-body">
+        <h5 className="card-title h5">{title}</h5>
+        {children}
+      </div>
+    </article>
+  );
+};
+
+export default function TestingCenter() {
+  return (
+    <React.Fragment>
+      <Head>
+        <title>Testing Centers | PrairieLearn</title>
+      </Head>
+      <PageBanner
+        title="Testing Centers"
+        subtitle="Delivering secure computer-based exams at scale"
+      />
+
+      <div className={classnames("container-fluid py-4")}>
+        <div className="container-md">
+          <div className="row align-items-center">
+            <div className="col-md-6 order-md-1 pt-4">
+              <div className="row justify-content-center">
+                <Heading className="text-center">
+                  What is a testing center?
+                </Heading>
+              </div>
+              <div className="row justify-content-center">
+                <p className={classnames("text-center", styles.larger_font)}>
+                  A testing center is a dedicated room equipped with computers
+                  and supervised by proctors, where students can take secure
+                  computer-based assessments.
+                </p>
+              </div>
+            </div>
+            <div className="col-md-6 order-md-2 pt-4 text-center">
+              <Image
+                src={cbtfFormatImage}
+                alt="testing center"
+                style={{
+                  maxWidth: "100%",
+                  height: "auto",
+                }}
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div className={classnames("container-fluid my-4")}>
+        <div className="container-md">
+          <div className="row">
+            <h4 className={classnames("py-3 ", styles.color, styles.center)}>
+              Benefits of a testing center
+            </h4>
+          </div>
+          <div className={styles.grid}>
+            <ValuePropCard title="The most secure exams">
+              Maximize test integrity with managed computers, lab firewall, and
+              vigilant proctoring. Students access only approved resources,
+              while cheating risks are minimized through computer screens, video
+              cameras and professional oversight.
```

Comment:
We use the Oxford comma elsewhere; let's be consistent (I'm open to changing our convention, but I'm personally partial to the Oxford comma)

```suggestion
              cameras, and professional oversight.
```